src/controllers/auth.controller.js

```diff
@@ -0,0 +1,107 @@
+import { ApiError } from '../exceptions/api.error.js';
+import { User } from '../modules/User.js';
```

Comment:
The import path for `User.js` is incorrect. The directory `modules` should be used instead of `moduls`.